cmd/cnbBuild.go

```diff
@@ -279,6 +290,14 @@ func runCnbBuild(config *cnbBuildOptions, telemetryData *telemetry.CustomData, u
 		}
 	}
 
+	telemetryData.Custom1Label = "cnbBuildStepData"
+	customData, err := json.Marshal(customTelemetryData)
```

Comment:
sending a json will not be helful here since it cannot be easily added to a report.

Please consider using one custom field per value.
Example:
https://github.com/SAP/jenkins-library/blob/master/cmd/artifactPrepareVersion.go#L102-L105

Reply:
Our idea was to collect all data in a form we can access, even if it means we need to post-process it to get meaningful results. We will likely be interested in more than five data points. We then have still four data points left that can be easily be part of a report.

Reply:
> sending a json will not be helful here..

But restricting to 5 data fields and miss out all other data, might not be helpful as well...

Reply:
@OliverNocon But would it even be possible to get the `json` out again, so that we can analyse it ourself?

Reply:
no, not in an easy way. 

We restricted it to 5 custom fields while having more data for each step.
This has been more than sufficient for all the other steps so far.

So answer for now, please use the ones which make most sense in your case.
If there is something which makes sense to other steps and is not there we could have a look to add those things to the common part.

Reply:
I don't know about the other steps, but there would be more than 5 data points we are interested in to understand our usage. But if that is not possible, I guess we restrict to the 2,3 most important ones, So we would already get some insight, but also keep the possibility to add something in the future.

Reply:
As discussed offline, we will go ahead with this approach (collecting a JSON object) as we are not sending this to SWA.